@AH-64D Apache Longbow/Addons/fza_ah64_ihadss/functions/fn_ihadssController.sqf

```diff
@@ -32,4 +32,4 @@ if (currentVisionMode player != 0 && cameraView != "GUNNER") then {
 [_heli] call fza_ihadss_fnc_CscopeDraw;
 [_heli] call fza_ihadss_fnc_constraintBoxDraw;
 [_heli] call fza_ihadss_fnc_steeringCursorDraw;
-
+[_heli] call fza_ihadss_fnc_routeCycle
```

Comment:
Why is this part of the ihadss? I think it was in the previous version, but that was a bad design choice and was done for no reason other than "the previous people had it there and I haven't rewritten it enough to correct it yet"

Could you have it as part of the `dms`? That is where I added some stub points for the few `route` functions we had already.